### DISCUSSION TITLE
Interpolated Fields - Public Variable Manager Modal

### DISCUSSION THREAD

**Discussion by swageblock** [mia-platform/community#573] [Ideas]
### Which area of this project could be most improved?

Console

### Description

It could be a benefit to the user experience to have interpolated Public Variables launch a modal to add or manage public variables.  

Basic Workflow: 
0. User enters interpolated value into text field (ex `/local/{{SOMEVALUE}}`)
1. UI detects interpolated value and presents a tooltip to enable Public Variable management
2. User clicks tooltip and is presented modal

![image](https://github.com/user-attachments/assets/07e05aa7-2e27-49a1-916c-f2b356b28470)

It seems beneficial for this modal to provide full management capabilities, but possibly displaying "Uses" of the public variable could be beneficial as well.

**Comment by fredmaggiowski** [comment]
Hi @swageblock thanks for the interesting proposal!
